File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex20.java
//Importando Scanner
import java.util.Scanner;

public class Ex20 {
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Categoria de um nadador\n\n");
		
		//vars
		int idade;
		
		//Entrada
		System.out.print("Insira a idade do nadador: ");
		idade = input.nextInt();
		
		//Processamento
		if ( idade>5 && idade<=7 )
			System.out.println("Categoria: Infantil");
		else if ( idade>=8 && idade<=10 )
			System.out.println("Categoria: Juvenil");
		else if ( idade>=11 && idade<=15 )
			System.out.println("Categoria: Adolescente");
		else if ( idade>=16 && idade<=30 )
			System.out.println("Categoria: Adulto");
		else if ( idade>30 )
			System.out.println("Categoria: Sênior");
		
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex20.c
#include <stdio.h>

main()
{
    printf("Idade de um nadador\n\n");

    //vars
    int idade;

    //Entrada
    printf("Insira a idade do nadaddor: ");
    scanf("%d",&idade);

    //Processamento e Saída
    if ( idade >= 5 && idade <= 7 )
        printf("Categoria: Infantil");
    else if ( idade >= 8 && idade <= 10 )
        printf("Categoria: Juvenil");
    else if ( idade >= 11 && idade <= 15 )
        printf("Categoria: Adolescente");
    else if ( idade >= 16 && idade <= 30 )
        printf("Categoria: Adulto");
    else
        printf("Categoria: Senior");

}
